watcher/watcher.go

```diff
@@ -155,7 +155,7 @@ func (w *Watcher) loadConfigurationDefaults() {
 	w.Config.SetDefault("watcher.autoScalingPeriod", 10)
 	w.Config.SetDefault("watcher.roomsStatusesReportPeriod", 10)
 	w.Config.SetDefault("watcher.ensureCorrectRoomsPeriod", 10*time.Minute)
-	w.Config.SetDefault("watcher.podStatesCountPeriod", 1*time.Minute)
+	w.Config.SetDefault("watcher.podStatesCountPeriod", 10*time.Second)
```

Comment:
Since this operation actually lists all pods form `GetPodMapFromRedis` (which uses an `HGETALL`) and parse them, could we put too much pressure into the Redis? My concern is that because of this constantly running other places where we use the same list pods would have some problem, WDYT? What about keeping this interval larger (for example `30s` or more)?